internal/server/statvar/golden/search_statvar/fem.json

```diff
@@ -85015,204 +85397,6 @@
           "dcid": "Percent_NeverOrOccasionally_ConsumptionOfVegetables_In_Count_Person_TertiaryEducation_Female"
         }
       ]
-    },
-    {
-      "dcid": "dc/g/Person_ActivityFrequency-NeverOrOccasionally_EducationalAttainment-UpperSecondaryEducationPostSecondaryNonTertiaryEducation_Gender-Female_HealthBehavior",
```

Comment:
This block of SVs seem deleted. Need to check if that's due to a limit on number of SVs chosen.